script.js

```diff
@@ -1,4 +1,66 @@
+//input = Username1, roll 2 dice, order of guess (dice1 or dice2)
+//output = Username1, output 2 guesses ... username2, output 2 guesses + outcome (who wins/lose)
+
+var gameMode = "enter username";
+var username = "";
+var myOutputValue = "";
+var winTrackerP1 = 0;
+var winTrackerP2 = 0;
+var numberOfTries = 0;
+
+var rollDice = function () {
+  var randomNumber = Math.floor(Math.random() * 7);
+  return randomNumber;
+};
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "‼ Invalid Entry ‼";
+  if (gameMode == "enter username") {
+    console.log("username entered");
+    username = input;
+    gameMode = "game start";
+    myOutputValue = `Welcome ${username} 👋, <br><br> Please press Submit to roll Dice. `;
+    console.log("dice game starts");
+    return myOutputValue;
+  }
+  if (gameMode == "game start") {
+    gameMode = "choose dice";
+    diceOne = rollDice();
+    diceTwo = rollDice();
+    dicePlayerOne = "" + diceOne + diceTwo;
```

Comment:
Hmm seems like you missed out declaring diceOne and diceTwo as variables 
e.g.` var diceOne = ... `

Same goes for dicePlayerOne